test/integration/guest_user_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class GuestUserTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test 'a guest user can view home page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.